src/utils/ReduxUtils.ts

```diff
@@ -1,7 +1,21 @@
 import * as _ from 'underscore';
+import * as Redux from 'redux';
+import { connect } from 'react-redux';
 
 export class ReduxUtils {
   static mergeProps(stateProps: any, dispatchProps: any, ownProps: any) {
     return _.extend({}, stateProps, dispatchProps, ownProps);
   }
 }
+
+export function ReduxConnect(mapStateToProps?: any, mapDispatchToProps?: any, mergeProps?: any, options?: any): (target: any) => any {
+  return target => (connect(mapStateToProps, mapDispatchToProps, mergeProps, options)(target) as any);
+}
+
+export interface IReduxAction<T> extends Redux.Action {
+  payload?: T;
+}
+
+export interface IReduxProps {
```

Comment:
As previously mentioned, we should avoid sending dispatch and letting the React component dispatch Redux actions. We should use mapDispatch to props and send  functions dispatching actions instead.